_episodes/01-introduction.md

```diff
@@ -3,13 +3,247 @@ title: "Introduction"
+
+![An example of a neural network](../fig/ML_DL_draft.png)
+
+
+### What sort of problems can deep learning solve?
+
+* Pattern/object recognition
+* Segmenting images (or any data)
+* Translating between one set of data and another, for example natural language translation.
+* Generating new data that looks similar to the training data, often used to create synthetic datasets, art or even "deepfake" videos.
+  * This can also be used to give the illusion of enhancing data, for example making images look sharper, video look smoother or adding colour to black and white images. But beware of this, its not an accurate recreation of the original data, but a recreation based on something statistically similar, effectively a digital imagination of what that data could look like.
+
+### What sort of problems can't deep learning solve?
+
+* Where only small amounts of training data is available.
+* Tasks requiring an explanation of how the answer was arrived at.
+* Being asked to classify things which are nothing like their training data.
+
+### What sort of problems can deep learning solve, but shouldn't be used for?
+
+Deep learning needs a lot of computational power, for this reason it often relies on specialist hardware like graphical processing units (GPUs). Many computational problems can be solved using less intensive techniques, but could still technically be solved with deep learning.
+
+The following could technically be achieved using deep learning, but it would probably be a very wasteful way to do it:
+
+* Logic operations, such as computing totals, averages, ranges etc. (see [this example](https://joelgrus.com/2016/05/23/fizz-buzz-in-tensorflow) applying deep learning to solve the "FizzBuzz" problem often used for programming interviews)
+* Modelling well defined systems, where the equations governing them are known and understood.
+* Basic computer vision tasks such as edge detection, decreasing colour depth or blurring an image.
+
+> ## Deep Learning Problems Exercise
+>
+> Which of the following could you apply deep learning to?
+> 1. Recognising whether or not a picture contains a bird.
+> 2. Calculating the median and interquartile range of a dataset.
+> 3. Identifying MRI images of a rare disease when only one or two example images available for training.
+> 4. Identifying people in pictures after being trained only on cats and dogs.
+> 5. Translating English into French.
+>
+> > ## Solution
+> >
+> > 1.  and 5 are the sort of tasks often solved with deep learning.
+> > 2. is technically possible but deep learning is massively overkill, you could do the same with much less computing power using traditional techniques.
+> > 3. will probably fail because there's not enough training data.
+> > 4. will fail because the deep learning system only knows what cats and dogs look like, it might accidentally classify the people as cats or dogs.
```

Comment:
Maybe add here (or in the explanation above) that it's a bit more nuanced: sometimes we can do transfer learning to finetune a network that was changed on a different, but similar task.